react-example/src/Screens/Login/Login.jsx

```diff
@@ -0,0 +1,94 @@
+// External modules
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+// Internal modules
+import './Login.less'
+import loginService from './login.service'
+import useLocalStorage from '../../Hooks/useLocalStorage/useLocalStorage'
+
+// Components
+import LayoutLogin from 'Components/LayoutLogin/LayoutLogin'
+
+
+export default function Login() {
+
+  // global state
+  const {item, saveItems: saveToken} = useLocalStorage('TOKEN-V1', [])
+  const navigate = useNavigate()
+
+  // Local state
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [camposCompletos, setCamposCompletos] = useState(false)
+
+
+  useEffect(() => {
+    if (item.success) {
+      navigate('/login-success')
+    }
+  }, [item])
+
+
+  // Methods
+  const onSubmit = async (event) => {
```

Comment:
Arrow functions with one parameter don't need the use of parenthesis, you can just do this:

`async event => {`